<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-PSQAKN1" name="Login_FF" tests="1" failures="1" timestamp="8 Feb 2017 11:21:14 GMT" time="0.541" errors="0">
  <testcase name="forgotPassword" time="0.512" classname="com.ch.occ.test.ForgottenPasswordTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.ch.occ.reports.TestCaseDetail.setErrorLogFile(TestCaseDetail.java:66)
	at com.ch.occ.reports.TestCaseDetail.error(TestCaseDetail.java:163)
	at com.ch.occ.test.ForgottenPasswordTest.forgotPassword(ForgottenPasswordTest.java:55)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- forgotPassword -->
</testsuite> <!-- Login_FF -->
